thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/spi/model/TimeSeries.java

```diff
@@ -53,17 +53,47 @@ public void addPredictedBaseline(DoubleSeries baselineValues) {
     this.df.addSeries(DataFrameUtils.COL_VALUE, baselineValues);
   }
 
+  /**
```

Comment:
Shouldn't the result include current time series too?

Reply:
Good point! We should include current as well.